draft-pda-core.md

```diff
@@ -166,24 +199,44 @@ limited set of aggregation functions, some of which we highlight below:
 This variety of aggregate types is sufficient to support a wide variety of
 data aggregation tasks.
 
-
-## Validating inputs
-
-A crucial task of any data collection pipeline is ensuring that the input data
-is valid. Going back to the example above, it's often useful to assert that each
-measurement is in a certain range, e.g., [0, 2^k) for some k. This straight-forward
-task is complicated in Prio by the fact that the inputs are secret shared. In
-particular, a malicious client can corrupt the computation by submitting random
-integers instead of a proper secret sharing of a valid input.
-
-To solve this problem, Prio introduces a light-weight zero-knowledge proof
-system designed to operate on secret shared data. In addition to its input
-share, each client sends to each aggregator a share of a "proof" of the input's
-validity. The aggregators use these proof shares in a protocol designed to
-establish the input's validity, without leaking the input itself. We describe
-this input-validation protocol in detail in [[TODO:citeme]].
-
-## Assembling Reports
+**Hits.**
+A common PA task that can't be solved efficiently with Prio is the
+`t`-*heavy-hitters* problem {{HITS}}. In this setting, each user is in
+possession of an `n`-bit string, and the goal is to compute the compute the set
+of strings that occur at least `t` times.
+
+[TODO: Provide an overview of the protocol of [BBG+21] and provide some
+intuition about how additive secret sharing is used. Be sure to introduce the
+notion of distributed point functions.]
+
+## Validating inputs in zero knowledge
+
+An essential task of any data collection pipeline is ensuring that the input
+data is "valid". Going back to the example above, it's often useful to assert
+that each measurement is in a certain range, e.g., `[0, 2^k)` for some `k`.
+This straight-forward task is complicated in our setting by the fact that the
+inputs are secret shared. In particular, a malicious client can corrupt the
+computation by submitting random integers instead of a proper secret sharing of
+a valid input.
+
+To solve this problem, each PA protocol in this document specifies a
+*zero-knowledge proof (ZKP) system* that allows the aggregators to verify that
```

Comment:
```suggestion
protocol that allows the aggregators to verify that
```

In general, I think we ought to try and keep these technical terms in the security considerations where possible.

Reply:
Again, I disagree. "ZKP" is no less technical of a term than "encryption", which we use freely in this section. It's just that "ZKP" is less common.

Reply:
I would split the difference here and say "includes a zero-knowledge proof". That is somewhat less formal than "zero-knowledge proof system" and I think clearer in this context.

Reply:
Or, for that matter, "proof"

Reply:
I reworked it to avoid using the word "system".